examples/README.md

```diff
@@ -0,0 +1,23 @@
+# Examples
+
+This folder contains different examples of Strimzi custom resources and demonstrates how they can be used.
+
+* [Kafka cluster examples](./kafka)
+    * Examples of Kafka deployments with different types of storage
+* [Kafka Connect examples](./kafka-connect)
+* [Kafka Connect Connector examples](./connector)
+* [Kafka Mirror Maker examples](./kafka-mirror-maker)
+* [Kafka Mirror Maker 2 examples](./kafka-mirror-maker2)
+* [Kafka Bridge examples](./kafka-bridge)
+* [Cruise Control and Kafka Rebalance examples](./cruise-control)
+* [User examples](./user)
+* [Topic examples](./topic)
+* [Prometheus metric examples](./metrics)
+    * Examples of Kafka components with enabled Prometheus metrics
+    * Grafana dashboards
+    * Prometheus Alert examples
+    * Sample Prometheus installation files
+    * Sample Grafana installation files
+    * JMX Trans examples
+* [Security examples](./security)
+    * Example deployments of Kafka, Kafka Connect and HTTP Bridge using TLS encryption, authentication and authorization
```

Comment:
if not with this PR we should have even Kafka Mirror Maker examples with authn and authz configured (and maybe even Cruise Control?)

Reply:
* As for Mirror Maker ... I was thinking about it. But I decided against it. It can be easily done with two Kafka clusters in one namespace and MM between them. But that is completely unrealistic since the clusters in reality would be running elsewhere and you cannot just mount the secrets out of the box. So I decided to skip it for the time being. We can add it later.
* As for Cruise Control - I though it is locked down out of the box. Or does it have some security settings?